tracing/otel/model/converter/converter.go

```diff
@@ -53,6 +53,21 @@ func ConvertSpans(spans []otelModels.Span, serviceName string) []jaegerModels.Sp
 			Warnings:      []string{},
 		}
 
+		// This is how Jaeger reports it
+		// Used to determine the envoy direction
+		atb_val := ""
+		if span.Kind == "SPAN_KIND_CLIENT" {
+			atb_val = "client"
+		} else {
+			if span.Kind == "SPAN_KIND_SERVER" {
```

Comment:
minor suggestion to make single `} else if {`.

Reply:
Updated. Thank you!